f5_openstack_agent/lbaasv2/drivers/bigip/agent.py

```diff
@@ -48,6 +54,7 @@
 
 LOG = oslo_logging.getLogger(__name__)
 
+# pzhang(NOTE): any uses? needs to check for later
```

Comment:
`any uses? needs to check for later` what does this mean?
Furthermore, I prefer to remove `pzhang(NOTE):`. 
1. Code readers don't care who you are.
2. You needn't be responsible to explain the code. 

However, it's up to you.